src/viam/sdk/robot/client.cpp

```diff
@@ -160,11 +162,8 @@ void RobotClient::refresh() {
     }
 
     std::unordered_map<Name, std::shared_ptr<Resource>> new_resources;
-    const RepeatedPtrField<ResourceName> resources = resp.resources();
-
     std::vector<ResourceName> current_resources;
-
-    for (auto& name : resources) {
+    for (const auto& name : resp.resources()) {
```

Comment:
Another `const`, but for some reason it required manual adjustment. I also got rid of the intermediate object since it wasn't needed.